src/pages/resultsView/enrichments/ExpressionEnrichmentsBoxPlot.tsx

```diff
@@ -0,0 +1,234 @@
+import * as React from 'react';
+import { observer } from "mobx-react";
+import { observable, computed } from 'mobx';
+import styles from "./styles.module.scss";
+import { MolecularProfile, Sample } from 'shared/api/generated/CBioPortalAPI';
+import {
+    ExpressionEnrichmentWithQ,
+    getAlterationsTooltipContent
+} from 'pages/resultsView/enrichments/EnrichmentsUtil';
+import * as _ from "lodash";
+import autobind from 'autobind-decorator';
+import { IBoxScatterPlotPoint, IStringAxisData, INumberAxisData, makeBoxScatterPlotData, getBoxPlotDownloadData } from '../plots/PlotsTabUtils';
+import BoxScatterPlot, { IBoxScatterPlotData } from 'shared/components/plots/BoxScatterPlot';
+import { remoteData, DownloadControls } from 'public-lib';
+import client from 'shared/api/cbioportalClientInstance';
+import LoadingIndicator from 'shared/components/loadingIndicator/LoadingIndicator';
+import { toConditionalPrecision } from 'shared/lib/NumberUtils';
+import { ExtendedAlteration } from '../ResultsViewPageStore';
+import { getSampleViewUrl } from 'shared/api/urls';
+import classNames from "classnames";
+import { getGeneSummary } from '../querySummary/QuerySummaryUtils';
+
+class MRNAEnrichmentsTabBoxPlot extends BoxScatterPlot<IBoxScatterPlotPoint> { }
+
+export interface IEnrichmentsBoxPlotProps {
+    selectedProfile: MolecularProfile;
+    groups: {
+        name: string,
+        description: string,
+        nameOfEnrichmentDirection?: string,
+        count: number,
+        color?: string,
+        samples:Pick<Sample, "uniqueSampleKey">[]
+    }[];
+    sampleKeyToSample: {
+        [uniqueSampleKey: string]: Sample;
+    }
+    queriedHugoGeneSymbols?: string[];
+    oqlFilteredCaseAggregatedData?: { [uniqueSampleKey: string]: ExtendedAlteration[] };
+    selectedRow?: ExpressionEnrichmentWithQ;
+}
+
+@observer
+export default class EnrichmentsBoxPlot extends React.Component<IEnrichmentsBoxPlotProps, {}> {
```

Comment:
sorry, I wasn't clear enough - I think `ExpressionEnrichmentsBoxPlot` is a good name for this. I was talking specifically about the class declared on line 23. I think it should be named more generically and I suggest `EnrichmentsBoxPlotComponent`

Reply:
done